docs/tutorial-local-server.rst

```diff
@@ -221,40 +95,17 @@ Both containers should be connected, and the heartbeat endpoint should only retu
 
     {"attachments":true, "cache":true, "permission":true, "signer": true, "storage":true}
 
-In the previous section we were using the ``main`` bucket directly, but in this setup, we will create the collections in the ``main-workspace`` bucket. Data will be automatically copied to the ``main-preview`` and ``main`` when requesting review and approving changes during the multi-signoff workflow.
-
-We'll use the same ``admin`` user:
+Unlike with *Simple Mode*, we'll need an ``admin`` user:
 
 .. code-block:: bash
 
     curl -X PUT ${SERVER}/accounts/admin \
          -d '{"data": {"password": "s3cr3t"}}' \
          -H 'Content-Type:application/json'
 
-The ``main-workspace`` bucket allows any authenticated user to create collections (like on STAGE):
-
-.. code-block:: bash
-
-    BASIC_AUTH=admin:s3cr3t
-
-    curl -X PUT ${SERVER}/buckets/main-workspace \
-         -d '{"permissions": {"collection:create": ["system.Authenticated"], "group:create": ["system.Authenticated"]}}' \
-         -H 'Content-Type:application/json' \
-         -u $BASIC_AUTH
-
-The ``main-preview`` and ``main`` buckets are (re)initialized with read-only permissions:
-
-.. code-block:: bash
-
-    curl -X PUT ${SERVER}/buckets/main-preview \
-         -d '{"permissions": {"read": ["system.Everyone"]}}' \
-         -H 'Content-Type:application/json' \
-         -u $BASIC_AUTH
+.. note::
 
-    curl -X PUT ${SERVER}/buckets/main \
-         -d '{"permissions": {"read": ["system.Everyone"]}}' \
-         -H 'Content-Type:application/json' \
-         -u $BASIC_AUTH
+    Another option is to use ``docker-compose`` with a cloned repository of https://github.com/mozilla/remote-settings
```

Comment:
Is it worth documenting that this is run with:
```
$ make build
$ docker-compose up
```

Reply:
Isn't `make build` supposed to be the same as `docker-compose build`?

Reply:
Not quite. If I just run `docker-compose build` in a fresh environment, I get this error:
```
 => ERROR [kinto:tests internal] load metadata for docker.io/library/remotesettings:build    
```

That's because `Dockerfile.Testing` starts 
```Dockerfile
FROM remotesettings:build
```

And since that container hasn't been built yet and it's not available via Dockerhub, it throws the error.

So, `make build` builds `remotesettings:build` first so that the testing container can be built on top of it. (It also goes ahead and builds the test container.)

We could alternatively instruct people to run the `web` container specifically with `docker-compose up web`.

Reply:
Gotcha. Will update the docs